src/main/java/juja/microservices/gamification/controller/AchievementController.java

```diff
@@ -26,6 +30,15 @@
 
     @RequestMapping(value = "/achieve/daily", method = RequestMethod.POST)
     @ResponseBody
+    @ApiOperation(
+            value = "Add points for daily report",
+            notes = "This method adds points for daily report"
+    )
+    //TODO @ApiImplicitParams doesn`t works
+    //    @ApiImplicitParams(value - @ApiImplicitParam(name, value, dataType, required))
```

Comment:
Please add values for these parameters. E.g, 
 @ApiImplicitParams(value = {@ApiImplicitParam( name = "email", value = "User's email", dataType = "application/json", required = true)})